lxd/instance_post.go

```diff
@@ -232,7 +232,7 @@ func instancePost(d *Daemon, r *http.Request) response.Response {
 		}
 
 		if targetMemberInfo == nil && s.GlobalConfig.InstancesPlacementScriptlet() != "" {
-			leaderAddress, err := d.gateway.LeaderAddress()
```

Comment:
interesting, could this previously have returned an error if not clustered?

Reply:
Apparentely yes: https://github.com/hamistao/lxd/blob/b98fa327315345e7927e65035e2cd8c7e63a8b5d/lxd/cluster/gateway.go#L618

Reply:
Shouldn't it stay that way though? If I understand this correctly, it seems pointless to excecute this bit in a standalone environment.

Reply:
Oh sorry, this entire section is gated with a `if target != "" && s.ServerClustered {`, so my comments here can be disregarded.

Reply:
Oh sorry, this entire section is gated with a if target != "" && s.ServerClustered {, so my comments here can be disregarded.